settings.php

```diff
@@ -32,6 +32,20 @@
         'tool_cloudwatch',
         get_string('pluginname', 'tool_cloudwatch')
     );
+
     $ADMIN->add('tool_cloudwatch_reports', $settings);
+
+    if ($ADMIN->fulltree) {
+
+        $settings->add(
+            new admin_setting_configselect(
```

Comment:
I'd like something more like the filters or auth admin settings, ie a table where each plugin can be toggled on and off independently

Reply:
use the auth plugins as the example it is better as its an admin_setting not its own admin page

Reply:
I have redone it this way.